thorlcr/thorutil/thbuf.cpp

```diff
@@ -1484,7 +1481,8 @@ class CSharedWriteAheadDisk : public CSharedWriteAheadBase
             mb.append((byte)0);
             size32_t len = mb.length();
             chunk.setown(getOutOffset(len)); // will find space for 'len', might be bigger if from free list
-            spillFileIO->write(chunk->offset, len, mb.toByteArray());
+            tempFileIO->write(chunk->offset, len, mb.toByteArray());
+            tempFileOwner->noteSize(highOffset);
```

Comment:
as per comment above. If there isn't one, we need a JIRA to add the basic spill(temp) stats.